packages/site/src/layout/NavMenu.tsx

```diff
@@ -151,11 +170,20 @@ export const StyledLink = ({
 export const StyledSubLink = ({
   to,
   children,
-}: PropsWithChildren<{ readonly to: string }>) => {
+  selectedRef,
+}: PropsWithChildren<{
+  readonly to: string;
+  readonly selectedRef: React.RefObject<HTMLAnchorElement>;
+}>) => {
+  const isSelected = location.pathname === to;
+
   return (
     <Link
       to={to ?? "/"}
-      className={`${styles.navMenuItem} ${styles.navMenuSubItem} ${styles.navMenuLink}`}
+      className={`${styles.navMenuItem} ${styles.navMenuSubItem} ${
```

Comment:
I think this one is long enough that it should be build up into a variable to make it a bit easier to read

Reply:
Created a helper function here 7265874edd19e3361206e4ad666fa2a5a81d443f